src/gluonts/torch/model/estimator.py

```diff
@@ -174,8 +174,8 @@ def train_model(
 
         validation_data_loader = None
 
-        with env._let(max_idle_transforms=max(len(training_data), 100)):
-            if validation_data is not None:
+        if validation_data is not None:
+            with env._let(max_idle_transforms=max(len(validation_data), 100)):
```

Comment:
Something I'm wondering, shouldn't there never be idle transforms, since the instance splitter always returns one window for validation per ts?